doc/mgmt/gnmi/SONiC_GNMI_Server_Interface_Design.md

```diff
@@ -441,18 +441,42 @@ For incremental configuration, 'config apply-patch' should not restart gNMI, bgp
 
 ##### 1.2.1.11.2 Full Configurations
 
-For full configuration, there’re 2 possible solutions: 
-* gNMI server needs to send response at first, and then invoke 'config reload'.
-
-<img src="images/full_rpc.svg" alt="full-gnmi" width="800px"/>
-
-* Use gNMI request and gNOI request together to implement full configuration update.
-
-<img src="images/full_rpc_gnoi.svg" alt="full-gnmi-gnoi" width="800px"/>
-
-The full configuration request will be overwritten by subsequent full configuration request or incremental configuration request.
-
-<img src="images/mixed requests.svg" alt="overwritten-config" width="800px"/>
+We will use single gnmi request to support full configuration update.
```

Comment:
during the full configuration update, do we block any config update request from other interfaces like REST, CLI?

Reply:
We do not block other configuration update requests. All configuration updates made before the 'config reload' will be overwritten, but those made after the 'config reload' will still function.